makefiles/util/checks.mk

```diff
@@ -0,0 +1,11 @@
+# Utilities to produce errors inside Make
+# Use this functions to produce friendlier error messages.
+
+# Produce an error if the value is empty
+#
+# Parameters
+#	value: a string which should not be empty
+#	message: The error message to display.
+# Returns:
+#	the first argument, if it is not zero.
```

Comment:
If it is not empty.

Reply:
d'oh!
```suggestion
#	the first argument, if it is not empty.
```

Reply:
Oh, wait, I should not be fixing this here, I should be doing it in #10439 !